recipes/asio/all/conanfile.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+from conans import ConanFile, tools
+
+
+class Asio(ConanFile):
+    name = "asio"
+    
+    url = "https://github.com/chriskohlhoff/asio"
```

Comment:
Why not https://github.com/conan-io/conan-center-index if the recipe is there?

Reply:
I always need someone to tell what is the meaning of those fields 😓